### PR TITLE
fix(nuxt): render head scripts that use `body: true`

### PR DESCRIPTION
**Pull request by harlan-zw**
### 🔗 Linked issue

@vueuse/head PR: https://github.com/vueuse/head/pull/67

issues: https://github.com/nuxt/framework/issues/2374, https://github.com/nuxt/framework/issues/5014, https://github.com/nuxt/framework/issues/3130

### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation or readme)
- [x] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

With nuxt using the latest version of @vueuse/head, it already has support for appending script and links to the end of the body tag with the `body: true` option. 

However due to a mismatch with property names of NuxtMeta and @vueuse/head nothing is rendered, this fixes that and adds some small doc and tests around the feature

Note: This does not work in components, this will need to be looked at in a separate PR as there is some extra complexity involved

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Review by pi0 — APPROVED** [review]

**Review by Atinux — APPROVED** [review]
Thanks you! Looks good to me, let's have a second eye from other core team members ☺️

File: docs/content/2.guide/2.features/4.head-management.md [hunk 1]
```diff
@@ -49,6 +49,28 @@ The `titleTemplate` can either be a string, where `%s` is replaced with the titl
 
 Now, if you set the title to `My Page` with `useHead` on another page of your site, the title would appear as 'My Page - Site Title' in the browser tab. You could also pass `null` to default to the site title.
 
+## Body Meta Tags
+
+::StabilityEdge{title="Body Meta Tatgs"}
```
Comment (DamianGlowala) [inline docs/content/2.guide/2.features/4.head-management.md:54]:
```suggestion
::StabilityEdge{title="Body Meta Tags"}
```

File: docs/content/2.guide/2.features/4.head-management.md [hunk 2]
```diff
@@ -49,6 +49,28 @@ The `titleTemplate` can either be a string, where `%s` is replaced with the titl
...
+::
+
+You can use the `body: true` option on the `link` and `script` meta tags to append them to the end of your `<body>` tag.
```
Comment (DamianGlowala) [inline docs/content/2.guide/2.features/4.head-management.md:57]:
```suggestion
You can use the `body: true` option on the `link` and `script` meta tags to append them to the end of the `<body>` tag.
```

File: docs/content/2.guide/2.features/4.head-management.md [hunk 3]
```diff
@@ -49,6 +49,28 @@ The `titleTemplate` can either be a string, where `%s` is replaced with the titl
...
+
+```vue
+<script setup>
+useHead({
+  script: [
+    {
+      src: 'https://third-party-script.com',
+      body: true
+    }
+  ],
```
Comment (DamianGlowala) [inline docs/content/2.guide/2.features/4.head-management.md:69]:
```suggestion
  ]
```

File: packages/nuxt/src/core/runtime/nitro/renderer.ts [hunk 4]
```diff
@@ -171,6 +171,7 @@ export default eventHandler(async (event) => {
       bodyAppend: normalizeChunks([
       `<script>window.__NUXT__=${devalue(ssrContext.payload)}</script>`,
       _rendered.renderScripts(),
+      // Note: bodyScripts may contain other tags other than <script>
```
Comment (DamianGlowala) [inline packages/nuxt/src/core/runtime/nitro/renderer.ts:174]:
```suggestion
      // Note: bodyScripts may contain tags other than <script>
```

**Review by DamianGlowala — CHANGES_REQUESTED** [review]

**Comment by pi0** [comment]
Thanks for docs improvements @DamianGlowala!

**Pull request merged**